{"name": "test_login01[case0]", "status": "broken", "statusDetails": {"message": "NameError: name 'act_res' is not defined", "trace": "self = <testcase.test_login.TestLogin object at 0x00000256600B7640>\ncase = <comon.excel_util.CaseData object at 0x00000256600CC370>\n\n    @pytest.mark.parametrize('case',ExcelUtil().read_excel('login'))\n    def test_login01(self, case):\n        print(case)\n        allure.attach(body=case.url, name='接口路径')\n        allure.attach(body=case.case_data, name='请求参数')\n        allure.attach(body=case.res, name='预期结果')\n        logger.info(\"*************** 开始执行用例 ***************\")\n        # logger.info('----------->开始执行测试案例:{}'.format(case['case_title']))\n        response = requests.get(url=case.url, data=eval(case.case_data))\n        # act_res=response.json()\n>       print(act_res)\nE       NameError: name 'act_res' is not defined\n\ntestcase\\test_login.py:26: NameError"}, "attachments": [{"name": "接口路径", "source": "e73c7fcb-040e-4c78-af28-ce61681b9015-attachment.attach"}, {"name": "请求参数", "source": "7b9cd0b1-bbb3-41f3-ae2f-b033bbb80903-attachment.attach"}, {"name": "预期结果", "source": "7504481e-c45c-4cb0-93ae-64f265eb3030-attachment.attach"}], "parameters": [{"name": "case", "value": "<comon.excel_util.CaseData object at 0x00000256600CC370>"}], "start": 1650292603108, "stop": 1650292603116, "uuid": "3251eb5d-4de0-4155-bd35-7ba8a95364c3", "historyId": "36f2f3d80fe1d5ee04c6e3c3e14bcb9c", "testCaseId": "f0a3740a932ee8262933e5b94534ef30", "fullName": "testcase.test_login.TestLogin#test_login01", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Mikey"}, {"name": "thread", "value": "15044-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_login"}]}